### DISCUSSION TITLE
Pokémon Legends: Arceus

### DISCUSSION THREAD

**Discussion by SultanOfficial49** [skyline-emu/skyline#554] [Game Requests]
Pokémon Legends: Arceus

**Comment by Kurosakidude** [comment]
Yup pls get it working along with sword shield and let's gos

**Comment by Nemomando** [comment]
Este é um game que não está acessível as pessoas, "o concorrente " não conseguiu fazer-lo rodar de formas satisfatória para snap860 se os senhores conseguirem fazer ele rodar no snapdragon 860 ou abaixo sem glitch já saíram na frente, en todo caso muito obrigado ao Desenvolvedores pelo empenho e graças a vocês que muitos podem preservar e conhecer jogos!muito obrigado

**Reply by PixelyIon** [reply]
@Nemomando Hey there, we appreciate it but I wanted to mention that this is an English-only forum as it is really hard to moderate otherwise. You should run your messages through [DeepL Translator](https://www.deepl.com/translator) to translate them from Portuguese to English, it is much better than Google Translate and almost always correct.

Translation: Olá, agradecemos, mas gostaria de mencionar que este é um fórum exclusivamente inglês, uma vez que é realmente difícil de moderar de outra forma. Deve passar as suas mensagens através do DeepL Tradutor para as traduzir de português para inglês, é muito melhor do que o Google Translate e quase sempre correcto.

**Comment by Tskk-dev** [comment]
It deeply concerns me that people want to actually run this

**Reply by finr0y** [reply]
for real bro pokemon players are something else 💀
